doc/source/getting_started/10min.rst

```diff
@@ -520,15 +520,15 @@ See the :ref:`Grouping section <groupby>`.
                       'D': np.random.randn(8)})
    df
 
-Grouping and then applying the :meth:`~DataFrame.sum` function to the resulting
+Grouping and then applying the :meth:`~pandas.core.groupby.GroupBy.sum` function to the resulting
```

Comment:
Is the tilde required here? Can you build this to see how this currently renders?

Reply:
There are no problems when I build the documentation locally. But when I pushed it to pandas-dev, some tests failed (previous commit). However, after I added tilde, all checks passed. I thought it might be because the link was displayed as pandas.core.groupby.GroupBy.sum() instead of sum().

Reply:
Does it actually render the link? This is different than how we link to GroupBy methods in our what’s new so not sure this would really solve the problem at hand (could be wrong)

Reply:
iirc the tilde is used to just print the last element in the link text. So, without the tilde the text link would be `pandas.core.groupby.GroupBy.sum`, and with the tilde should be just `sum`.

I don't think the doc build should fail in any of the cases. May be the error was something else?

Reply:
> Does it actually render the link?

Yes, it does render the link with and without the tilde.

> iirc the tilde is used to just print the last element in the link text

Yes, it is correct. In the last commit the link is displayed as `sum()` instead of `pandas.core.groupby.GroupBy.sum`.

> I don't think the doc build should fail in any of the cases. May be the error was something else?

I checked today locally, there were no problems with building of the documentation, so the error might be not related.

Should I push it without tilde? Or should I leave it like it is?